mountpoint-s3/src/cli.rs

```diff
@@ -918,6 +922,15 @@ fn validate_mount_point(path: impl AsRef<Path>) -> anyhow::Result<()> {
     Ok(())
 }
 
+#[cfg(feature = "sse_kms")]
+fn validate_sse_args(sse_type: &Option<String>, sse_kms_key_id: &Option<String>) -> anyhow::Result<()> {
+    if sse_kms_key_id.is_some() && *sse_type == Some("AES256".to_owned()) {
+        Err(anyhow!("--sse_kms_key_id can not be used with AES256 sse_type"))
```

Comment:
nit: use the actual flag
```suggestion
        Err(anyhow!("--sse_kms_key_id can not be used with --sse AES256"))
```

Reply:
nit: it's actually `--sse-kms-key-id` (clap converts _ to - in arguments by default)

Reply:
nice catch, fixed the error message